qbatch/qbatch.py

```diff
@@ -750,6 +759,10 @@ def qbatchParser(args=None):
         "--shell", default=SHELL,
         help="""Shell to use for spawning jobs
         and launching single commands""")
+    group.add_argument(
+        "--block", action="store_true",
+        help="""For SGE, PBS and SLURM, blocks execution until jobs are finished.
+        'sync -y' is called for SGE, '-Wblock=true' for PBS and '--wait' for SLURM.""")
```

Comment:
We don't need these details here.

Reply:
Let me know if the update is satisfying there.

Reply:
The `o_block` definitions should be the inline assignment type like
https://github.com/pipitone/qbatch/blob/master/qbatch/qbatch.py#L448

Currently, if block is undefined, o_block will not exist, and the template filling will fail.

Reply:
This should be addressed with the latest commit.